get-dependency-list.js

```diff
@@ -22,12 +24,45 @@ module.exports = function(filename, serverless) {
   const modulesToProcess = [];
   const localFilesToProcess = [filename];
 
+  const shouldUseLocalNodeModules = lodash.get(serverless.service.custom, 'serverless-plugin-include-dependencies.shouldUseLocalNodeModules', false);
```

Comment:
```suggestion
  const shouldUseLocalNodeModules = !!serverless.service.custom?.['serverless-plugin-include-dependencies']?.shouldUseLocalNodeModules;
```

can we drop the `lodash`?

Reply:
Basically we can, I think it's more elegant + it has fail-safe in case serverless.service and serverless.service.custom are undefined

Reply:
also in my suggestion is safe due to the optional changing operator.

Reply:
fixed